.github/workflows/.coderabbit.yaml

```
@@ -18,14 +18,65 @@ path_instructions:
+
+  # Test files (to ensure tests cover security scenarios)
+  - path: "**/sandbox/**/*.spec.ts"
+    instructions: "TEST SECURITY: Ensure tests cover autoDeleteInterval = -1 and autoStopInterval = 0 scenarios. Verify tests validate that disabled values never trigger automated actions."
+    
+  - path: "**/sandbox/**/*.test.ts"
+    instructions: "TEST SECURITY: Ensure tests cover autoDeleteInterval = -1 and autoStopInterval = 0 scenarios. Verify tests validate that disabled values never trigger automated actions."
 
 chat:
   auto_reply: true
 
 ignore:
```

actionlint (1.7.7)

unexpected key "ignore" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)